src/vendor/wordpress/demo/image.twig

```diff
@@ -0,0 +1,30 @@
+<div
+  class="demo-container"
```

Comment:
Consider using [the Rhythm object pattern](https://v-next--cloudfour-patterns.netlify.app/?path=/docs/objects-rhythm--default-story) for this, especially since your styles account for it.